### PR TITLE
Updated User Guide Styling

### PR DESCRIPTION
**Pull request by ritvi12**
### Description

Formatted the styling of the UG. 
Changed the language of the UG to be catered to the target audience.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -133,69 +137,79 @@ Adds a person to the address book.
...
+**Warning**: After confirmation, any deleted data or patient contact cannot be retrieved. This action is IRREVERSIBLE.
 
 </box>
 
-* Deletes the person at the specified `INDEX` or with the specified `NAME`.
-* The name refers to the full name as shown in the displayed person list.
-* The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
-* The name provided must be the person's full name.
+* Deletes the patient contact at the specified `INDEX` or with the specified `NAME`.
```
Comment (Bonzzz3) [inline docs/UserGuide.md:196]: Same as above

File: docs/UserGuide.md [hunk 2]
```diff
@@ -133,69 +137,79 @@ Adds a person to the address book.
...
+<box type="definition" icon=":fa-solid-spell-check:" light no-background>
 
 **Format**: `delete INDEX` / `delete NAME`
+</box>
 
-<box type="warning" seamless>
+<box type="warning" icon=":fa-solid-circle-info:" light no-background>
 
-**Warning**: After confirmation, any deleted data or contact cannot be retrieved. This action is IRREVERSIBLE.
+**Warning**: After confirmation, any deleted data or patient contact cannot be retrieved. This action is IRREVERSIBLE.
```
Comment (Bonzzz3) [inline docs/UserGuide.md:192]: I think patient's contact sounds better. But this is fine too if patient contact as a whole is treated as a thing

File: docs/UserGuide.md [hunk 3]
```diff
@@ -133,69 +137,79 @@ Adds a person to the address book.
...
+* Deletes the patient contact at the specified `INDEX` or with the specified `NAME`.
+* The `NAME` refers to the full name as shown in the displayed patient contact list.
+* The `INDEX` refers to the index number shown in the displayed patient contact list.
+* The `INDEX` **must be a positive integer** 1, 2, 3, …​
+* The `NAME` provided must be the person's full name.
+<box type="important" light no-background>
 
 **Confirmation Requirement:**
 
 * A confirmation window will appear before deleting a person, asking you to confirm the deletion.
```
Comment (Bonzzz3) [inline docs/UserGuide.md:205]: Maybe can edit this to be deleting a person's contact for consistency

**Review by Bonzzz3 — APPROVED** [review]
LGTM, just a few suggestions

File: docs/UserGuide.md (hunk 2 above)
Comment (ritvi12) [reply docs/UserGuide.md:192]: I think patient contact as a whole is better cause I meant it as a whole list

File: docs/UserGuide.md (hunk 1 above)
Comment (ritvi12) [reply docs/UserGuide.md:196]: I can change this one to be patient's contact

File: docs/UserGuide.md [hunk 4]
```diff
@@ -133,69 +137,79 @@ Adds a person to the address book.
...
+</box>
 
 [Return to Table of Contents](#medicontact)
-
 <br>
 <br>
 
 
-### Deleting a person : `delete`
+### <i class="fa-solid fa-user-slash"></i> Deleting a person : `delete`
```
Comment (myrilla12) [inline docs/UserGuide.md:181]: Can consider replacing "person" with "patient" in the heading

File: docs/UserGuide.md [hunk 5]
```diff
@@ -133,69 +137,79 @@ Adds a person to the address book.
...
-* Deletes the person at the specified `INDEX` or with the specified `NAME`.
-* The name refers to the full name as shown in the displayed person list.
-* The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
-* The name provided must be the person's full name.
+* Deletes the patient contact at the specified `INDEX` or with the specified `NAME`.
+* The `NAME` refers to the full name as shown in the displayed patient contact list.
+* The `INDEX` refers to the index number shown in the displayed patient contact list.
+* The `INDEX` **must be a positive integer** 1, 2, 3, …​
+* The `NAME` provided must be the person's full name.
```
Comment (myrilla12) [inline docs/UserGuide.md:200]: Can consider replacing "person's" with "patient's"

File: docs/UserGuide.md (hunk 3 above)
Comment (myrilla12) [inline docs/UserGuide.md:205]: Can replace "person" with "patient"

File: docs/UserGuide.md [hunk 6]
```diff
@@ -210,41 +224,47 @@ Deletes the specified person from the address book.
...
 <img src="images/deleteCommand_output.png" width="800" style="border: 2px solid lightgrey; border-radius: 10px;">
+</box>
 
 [Return to Table of Contents](#medicontact)
-
 <br>
 <br>
 
-### Editing a person : `edit`
+### <i class="fa-solid fa-pen"></i> Editing a person : `edit`
```
Comment (myrilla12) [inline docs/UserGuide.md:233]: Can replace "person" with "patient" in this heading too

File: docs/UserGuide.md [hunk 7]
```diff
@@ -255,31 +275,32 @@ Edits an existing person in the address book.
...
 
 [Return to Table of Contents](#medicontact)
-
 <br>
-
 <br>
 
 
-### Filtering persons by age group or appointment dates: `filter`
+### <i class="fa-solid fa-filter"></i> Filtering persons by age group or appointment dates: `filter`
```
Comment (myrilla12) [inline docs/UserGuide.md:297]: Can replace "persons" with "patients"

File: docs/UserGuide.md [hunk 8]
```diff
@@ -313,33 +336,39 @@ Filter persons whose age and/or appointment dates are within the specified range
...
 <img src="images/filterCommand_output.png" width="800" style="border: 2px solid lightgrey; border-radius: 10px;">
+</box>
 
 [Return to Table of Contents](#medicontact)
-
 <br>
 <br>
 
-### Locating persons by name or phone number: `find`
+### <i class="fa-solid fa-magnifying-glass"></i> Locating persons by name or phone number: `find`
```
Comment (myrilla12) [inline docs/UserGuide.md:345]: Can replace "persons" with "patients"

File: docs/UserGuide.md [hunk 9]
```diff
@@ -487,68 +523,81 @@ The following is an example of a valid JSON file content.
...
 
 [Return to Table of Contents](#medicontact)
-
 <br>
-
 <br>
 
 
-### Listing all persons : `list`
+### <i class="fa-solid fa-list"></i> Listing all persons : `list`
```
Comment (myrilla12) [inline docs/UserGuide.md:530]: Can replace "persons" with "patients"

File: docs/UserGuide.md [hunk 10]
```diff
@@ -487,68 +523,81 @@ The following is an example of a valid JSON file content.
...
+</box>
 
 [Return to Table of Contents](#medicontact)
-
 <br>
 <br>
 
 
-### Listing starred persons : `list *`
+### <i class="fas fa-clipboard-list"></i> Listing starred persons : `list *`
```
Comment (myrilla12) [inline docs/UserGuide.md:560]: Can replace "persons" with "patients"

File: docs/UserGuide.md [hunk 11]
```diff
@@ -487,68 +523,81 @@ The following is an example of a valid JSON file content.
...
 <img src="images/listStarCommand_output.png" width="800" style="border: 2px solid lightgrey; border-radius: 10px;">
+</box>
 
 [Return to Table of Contents](#medicontact)
-
 <br>
 <br>
 
-### Editing a person's notes : `note`
+### <i class="fa-solid fa-note-sticky"></i> Editing a person's notes : `note`
```
Comment (myrilla12) [inline docs/UserGuide.md:588]: Can replace "person's" with "patient's"

File: docs/UserGuide.md [hunk 12]
```diff
@@ -565,6 +614,7 @@ Edits an existing person's note in the address book, which contains `PREVIOUS AP
 - You can remove all the person’s remarks by typing `r/` without specifying any remarks after it.
```
Comment (myrilla12) [inline docs/UserGuide.md:614]: Can replace "person's" with "patient's"

File: docs/UserGuide.md [hunk 13]
```diff
@@ -609,22 +664,26 @@ Sorts the persons in the address book on the basis of their earliest appointment
...
 <img src="images/sortCommand_output.png" width="800" style="border: 2px solid lightgrey; border-radius: 10px;">
+</box>
 
 [Return to Table of Contents](#medicontact)
-
 <br>
 <br>
 
-### Adding a person to favourites list : `star`
+### <i class="fa-solid fa-star"></i> Adding a person to favourites list : `star`
```
Comment (myrilla12) [inline docs/UserGuide.md:673]: Can replace "person" with "patient"

File: docs/UserGuide.md [hunk 14]
```diff
@@ -639,23 +698,27 @@ Stars the specified person from the address book into favourites.
...
+</box>
 
 [Return to Table of Contents](#medicontact)
-
 <br>
 <br>
 
 
-### Removing a person from favourites list : `unstar`
+### <i class="fa-regular fa-star"></i> Removing a person from favourites list : `unstar`
```
Comment (myrilla12) [inline docs/UserGuide.md:708]: Can replace "person" with "patient"

File: docs/UserGuide.md [hunk 15]
```diff
@@ -670,24 +733,26 @@ Unstars the specified person from the address book removing them from favourites
...
 <img src="images/unstarCommand_output.png" width="800" style="border: 2px solid lightgrey; border-radius: 10px;">
+</box>
 
 [Return to Table of Contents](#medicontact)
-
 <br>
 <br>
 
-### Viewing a person's notes : `view`
+### <i class="fa-solid fa-eye"></i> Viewing a person's notes : `view`
```
Comment (myrilla12) [inline docs/UserGuide.md:742]: Can replace "person's" with "patient's"

**Review by myrilla12 — CHANGES_REQUESTED** [review]
Changes look good! There are a few spots whereby "person" has not been replace with "patient" though, mainly in the headings.

**Comment by ritvi12** [comment]
All changes made

**Review by myrilla12 — APPROVED** [review]
LGTM

**Pull request merged**